TECH-OVERSIGHT-COMMITTEE.md

```diff
@@ -92,11 +92,13 @@ during the community feedback period at the end of each meeting.
 
 The current members of the TOC are shown below.
 
-| &nbsp;                                                         | Member         | Company | Profile                                              | Term Start | Term End |
-| -------------------------------------------------------------- | -------------- | ------- | ---------------------------------------------------- | ---------- | --------
-| <img width="30px" src="https://github.com/evankanderson.png">  | Evan Anderson  | VMware  | [@evankanderson](https://github.com/evankanderson)   | Bootstrap  | 2021     |
-| <img width="30px" src="https://github.com/markusthoemmes.png"> | Markus Thömmes | Red Hat | [@markusthoemmes](https://github.com/markusthoemmes) | 2020-06-09 | 2022     |
-| <img width="30px" src="https://github.com/rhuss.png">          | Roland Huß     | Red Hat | [@rhuss](https://github.com/rhuss)                   | 2021-02-16 | 2022     |
+| &nbsp;                                                         | Member           | Company | Profile                                              | Term Start | Term End |
+| -------------------------------------------------------------- | ---------------- | ------- | ---------------------------------------------------- | ---------- | --------
+| <img width="30px" src="https://github.com/dprotaso.png">       | Dave Protasowski | VMware  | [@dprotaso](https://github.com/dprotaso)             | 2021-05-26 | 2022     |
+| <img width="30px" src="https://github.com/evankanderson.png">  | Evan Anderson    | VMware  | [@evankanderson](https://github.com/evankanderson)   | Bootstrap  | 2023     |
```

Comment:
Since this is technically a new term for Evan, should we update Bootstrap to this election date?

Reply:
@geekygirldawn wdyt?